frontend/src/api.ts

```diff
@@ -308,11 +319,17 @@ export const mediaUploadGet = (item: IMediaResource) : Promise<IMediaUploadResou
   return apiFetch(API_ENDPOINTS.mediaList + item.id + '/upload');
 };
 
+/** Retrieve a media resource's analytics. */
+export const analyticsGet = (id: string) : Promise<IAnalyticsResponse> => {
+  if (ANALYTICS_FROM_PAGE) { return Promise.resolve(ANALYTICS_FROM_PAGE); }
```

Comment:
Could this use ``RESOURCES_FROM_PAGE`` instead? Then one can get rid of the ``analytics.html`` template.

Reply:
We may want to change ``resourceFromPageById`` to ``resourceFromPageByPath`` and pass the path to the resource rather than an id.